services/src/main/java/org/keycloak/services/resources/admin/AuthenticationManagementResource.java

```diff
@@ -497,6 +499,19 @@ public Response addExecutionToFlow(@PathParam("flowAlias") String flowAlias, Map
 
         execution = realm.addAuthenticatorExecution(execution);
 
+        if (f instanceof ConfiguredProvider) {
+            ConfiguredProvider internalProviderFactory = (ConfiguredProvider) f;
+            AuthenticatorConfigModel config = internalProviderFactory.getConfig();
+
+            if (config != null) {
+                // creates a default configuration if the factory defines one
+                // useful for internal providers that already provide a built-in configuration
+                AuthenticatorConfigRepresentation configRepresentation = ModelToRepresentation.toRepresentation(
+                        config);
+                newExecutionConfig(execution.getId(), configRepresentation);
```

Comment:
Should we close the response created by this method call?

Reply:
Sorry, which response should we close?

Reply:
Ahh right. Will close it.